apps/src/codebridge/FileBrowser/prompts/openImportFromBackpackPrompt.tsx

```diff
@@ -159,6 +171,7 @@ export const openImportFromBackpackPrompt = async ({
               handleConfirm: () =>
                 fetchFileContentAndProcess(
                   selectedBackpackFileName,
+                  EVENTS.CODEBRIDGE_IMPORT_FROM_BACKPACK_REPLACE,
```

Comment:
I think this should be `RENAME` since it's duping a support file name.

Reply:
good catch!